packages/eslint-plugin/src/rules/no-unsafe-enum-comparison.ts

```diff
@@ -66,6 +66,19 @@ export default util.createRule({
       'BinaryExpression[operator=/=|<|>/]'(
```

Comment:
```suggestion
      'BinaryExpression[operator=/^[=<>]$/]'(
```

...Much simpler?

Reply:
Thanks for the suggestion, please see [3d812c6](https://github.com/typescript-eslint/typescript-eslint/pull/7074/commits/3d812c69909794f8ede28c717cf86fbae5cabf63), I've replaced the regex with `^[<>!]?={0,3}$` instead to also match `!==` and `===`